.circleci/config.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:7.10
```

Comment:
Can you upgrade the node version to the most stable build of `node` - v12?
This is to ensure there are no flaky tests or weird bugs while running tests in `circleci`.